include/geolib/math_types.h

```diff
@@ -745,11 +745,30 @@ class Transform3T {
     void setRPY(T roll, T pitch, T yaw)  {
         R.setRPY(roll, pitch, yaw);
     }
+    
+    void getRPY(T &roll, T &pitch, T& yaw){
```

Comment:
```suggestion
    void getRPY(T& roll, T& pitch, T& yaw) const {
```

Reply:
I have really no clue how a Transform of an int or unsigned should look. But I don't  think it's RPY, will be an int/unsigned. Also this conflicts with `getYaw`, which always returns a double.

Reply:
True, however, I wanted to keep the input/output consistent with ` void setRPY(T roll, T pitch, T yaw) ` above

Reply:
Lets change both.

Reply:
done